sklearn/ensemble/tests/test_forest.py

```diff
@@ -1149,7 +1149,7 @@ def check_memory_layout(name, dtype):
     y = iris.target
```

Comment:
In this file we need a test to check the deprecation warning of `base_estimator_` fitted attribute.